ironfish/src/mining/poolDatabase/database.test.ts

```diff
@@ -0,0 +1,43 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import { Config } from '../../fileStores'
+import { NodeFileProvider } from '../../fileSystems'
+import { createRootLogger } from '../../logger'
+import { getUniqueTestDataDir } from '../../testUtilities/utils'
+import { PoolDatabase } from './database'
+
+describe('poolDatabase', () => {
+  let db: PoolDatabase
+
+  beforeEach(async () => {
+    const logger = createRootLogger().withTag('test')
+    const dataDir = getUniqueTestDataDir()
```

Comment:
Do we need to do any cleanup on these?

Reply:
Based on a bit of poking around, I don't think we do any cleanup on test data on a test-run. Instead, it cleans it up on your _next_ test run. So you always have 1 run of data laying around. It'd be worth cleaning that up in another PR at some point